modules/monitoring-granting-users-permissions-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="granting-users-permissions-for-core-platform-monitoring_{context}"]
+= Granting users permissions for core platform monitoring
+
+As a cluster administrator, you can monitor all core {product-title} and user-defined projects.
+
+You can also grant developers and other users different permissions for core platform monitoring:
+
+* To monitor core platform monitoring components.
+* To access the Thanos Querier API endpoints.
+* To manage alert silences.
+* To monitor the Alertmanager API.
```

Comment:
(discussion) I feel that we mix and match high-level functionality ("monitor core platform monitoring components") with low-level stuff ("access Thanos Querier API"). Should we avoid implementation details and leave that for the role descriptions?

Reply:
Yes, that makes sense. I tried to create something similar to what is in the UWM section, but here it is a bit awkward. I aggree that it would be probably better to remove it :+1:  Thank you!